src/NServiceBus.Core.Tests/ApprovalFiles/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2056,13 +2056,19 @@ namespace NServiceBus.Pipeline
     }
     public interface IRecoverabilityActionContext : NServiceBus.Extensibility.IExtendable, NServiceBus.ICancellableContext, NServiceBus.Pipeline.IBehaviorContext
     {
-        NServiceBus.Transport.ErrorContext ErrorContext { get; }
+        System.Exception Exception { get; }
+        NServiceBus.Transport.IncomingMessage FailedMessage { get; }
+        int ImmediateProcessingFailures { get; }
```

Comment:
What about [`DelayedDeliveriesPerformed`](https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Transports/ErrorContext.cs#L61)?

Reply:
We are not using it and its also pulled from a header so it didn't feel needed? https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Recoverability/DelayedRetriesHeaderExtensions.cs#L8

no strong opinion tho

Reply:
It's actually pulled in the ctor so this value isn't coming from the transport https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Transports/ErrorContext.cs#L39

Reply:
I think you are right, I found a place where we are using it incorrectly so I exposed it and changed the code to use to use it, see the last commit